src/routes/+page.svelte

```diff
@@ -5,51 +5,54 @@
 	import Actions from '$lib/components/Actions.svelte'
 	import { galleryStore } from '$lib/store'
 	import { Slider } from '$lib/components/ui/slider'
-	import type { ImageType } from '$lib/utils/types'
+	import { open } from '@tauri-apps/api/dialog'
+	import { invoke, convertFileSrc } from '@tauri-apps/api/tauri'
 	import TagsList from '$lib/components/TagsList.svelte'
-	import { toast } from 'svelte-sonner'
-
-	let images: ImageType[] = []
-	let zoomLevel: number[] = [$galleryStore.zoomLevel]
 
 	const uploadFolder = async () => {
-		images = []
-		$galleryStore.images = []
-		const filesDir: File[] = (await openDirectory()) as File[]
+		let selected = await open({
```

Comment:
I tested the app on the browser, it doesn't open the dialog on the browser, will test it on the Tauri build app and update the comment.

Reply:
@alvinosh 
Tested on Mac too, It doesn't work on the browser.
And for the Tauri app, it seems it works for only file selection, not for the whole folder.

cc: @kirillt

Reply:
@prokawsar Functioning in browser is less priority than in Tauri. Convenient for development, but we need only standalone app.

@alvinosh In Linux, I can select multiple files (cannot folder), but then nothing happens. Images don't load.

Reply:
@prokawsar Im pretty sure when working with tauri the only reason there is a localhost running is for hot reloading duirng development purposes, you are supposed to do all your testing in the desktop application.

Reply:
@kirillt Folder not being able to be selected is expected behaviour (the allowFolder option is not set accidentally). But images should load, is there any errors in the console?